### PR TITLE
37545

### PR DESCRIPTION
**Pull request by chandragj**
|Field|Value|
|-----|-----|
| Redmine Ticket Number | [37545](https://devops.dchbx.org/redmine/issues/37545) |
| App-Dev Road Map # | n/a |
| Start Date | n/a |
| Due Date | 12/14/2018  |
| App-Dev Priority | UP-Disqueued |
| Development Story Points | 0.5 |
| Peer Review Story Points | 0.5 |
| Ticket Author | @karashearer5 |

### CODE REVIEW

File: app/models/person.rb [hunk 1]
```diff
@@ -461,8 +461,8 @@ def verification_type_by_name(type)
 # collect all ridp_verification_types user in case of unsuccessful ridp
   def ridp_verification_types
     ridp_verification_types = []
-    ridp_verification_types << 'Identity' if consumer_role  && !consumer_role.person.completed_identity_verification?
-    ridp_verification_types << 'Application' if consumer_role && !consumer_role.person.completed_identity_verification?
+    ridp_verification_types << 'Identity'
```
Comment (saipraveen18) [inline app/models/person.rb]: check for consumer role

File: app/views/insured/interactive_identity_verifications/_outstanding_ridp_documents.html.erb [hunk 2]
```diff
@@ -14,13 +14,19 @@
...
+                       <% families = controller_name == 'families' %>
+                      <div class='v-type-status <%= families ? "col-md-2" : "col-md-3" %>'>
                       <span class="label lb-md label-<%=ridp_type_class(ridp_type, @person)%>" id="status">
                         <%= show_ridp_type(ridp_type, @person) %>
                       </span>&nbsp;
                         <%= pundit_span Family, :updateable? %>
                       </div>
-                      <div class="v-type-upload col-md-4">
+                      <%if families %>
+                        <div class="v-type-due-date col-md-3">
```
Comment (saipraveen18) [inline app/views/insured/interactive_identity_verifications/_outstanding_ridp_documents.html.erb]: what is the purpose of this div tag?

Comment (chandragj) [reply app/views/insured/interactive_identity_verifications/_outstanding_ridp_documents.html.erb]: To align upload Documents button and action menu, div tag was written.

File: app/models/person.rb (hunk 1 above)
Comment (chandragj) [reply app/models/person.rb]: Added Consumer role

**Pull request closed without merging**